app/src/main/java/chat/rocket/android/RocketChatApplication.java

```diff
@@ -23,6 +23,7 @@
   public void onCreate() {
     if (BuildConfig.DEBUG) {
```

Comment:
They should have a NoOp version like leakcanary does (https://github.com/square/leakcanary)

we will need to isolate all the Setho code and have separate implementations for `debug` and `release`